crates/sdk/src/args.rs

```diff
@@ -276,7 +276,9 @@ pub struct TxTransparentTransfer<C: NamadaTypes = SdkTypes> {
     /// Common tx arguments
     pub tx: Tx<C>,
     /// The transfer specific data
-    pub data: Vec<TxTransparentTransferData<C>>,
+    pub sources: Vec<TxShieldingTransferData<C>>,
```

Comment:
I think we can remove the `TxTransparentTransferData` here above right?

Reply:
Oh yes, I forgot to remove this. Thanks!